dataikuapi/utils.py

```diff
@@ -94,3 +94,65 @@ def str_to_bool(s):
                                                 doublequote=True):
                 yield [none_if_throws(caster)(val)
                         for (caster, val) in dku_zip_longest(casters, uncasted_tuple)]
+
+class DSSExtendableDict(dict):
```

Comment:
- I think the correct word is `Extensible`
- Could use a docstring specifying in which way this is or isn't compatible with `dict`